app/src/main/java/com/example/trainingintive/util/Extensions.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.trainingintive.util
+
+import io.reactivex.rxjava3.disposables.CompositeDisposable
+import io.reactivex.rxjava3.disposables.Disposable
+
+operator fun CompositeDisposable.plusAssign(disposable: Disposable?) {
```

Comment:
Good job that you implemented a custom operator! There is also a library called "RxKotlin" and it contains this operator. But it's okay to implement it at it's own if you are not using any more things from this library :).